src/main/scala/org/ergoplatform/nodeView/state/StateConstants.scala

```diff
@@ -13,9 +13,9 @@ import scorex.crypto.authds.ADDigest
   * @param settings          - node settings
   */
 case class StateConstants(nodeViewHolderRef: Option[ActorRef], settings: ErgoSettings) {
-  lazy val emission: EmissionRules = settings.emission
-  lazy val genesisEmissionBox: ErgoBox = ErgoState.genesisEmissionBox(emission)
+  lazy val emission: EmissionRules = settings.chainSettings.emission
+  lazy val genesisEmissionBox: ErgoBox = ErgoState.genesisBoxes(settings.chainSettings).head
```

Comment:
@catena2w looks like we creating `genesisBoxes` more than once. Here and in UTXO genesis state.

Reply:
fixed